v20.1/temporary-tables.md

```diff
@@ -0,0 +1,211 @@
+  vehicles   | CREATE TEMP TABLE vehicles (
+             |     id UUID NOT NULL,
+             |     city STRING NOT NULL,
+             |     type STRING NULL,
+             |     owner_id UUID NULL,
+             |     creation_time TIMESTAMP NULL,
+             |     CONSTRAINT "primary" PRIMARY KEY (city ASC, id ASC),
+             |     INDEX vehicles_auto_index_fk_city_ref_users (city ASC, owner_id ASC),
+             |     FAMILY "primary" (id, city, type, owner_id, creation_time)
+             | )
+(1 row)
+~~~
+
+To show all temp tables in a session's temporary schema, use `SHOW TABLES FROM pg_temp`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW TABLES FROM pg_temp;
+~~~
+
+~~~
+  table_name
+--------------
+  users
+  vehicles
+(2 rows)
+~~~
+
+You can also use the full name of the temporary schema in the `SHOW` statement (e.g., `SHOW TABLES FROM pg_temp_1584540651942455000_1`).
+
+{{site.data.alerts.callout_info}}
+The [`SHOW TABLES`](show-tables.html) statement defaults to the `public` schema, which doesn't include temp tables. As a result, using `SHOW TABLES` without specifying a schema won't return any temp tables.
+{{site.data.alerts.end}}
+
+Although temp tables are not included in the `public` schema, metadata for temp tables is included in the [`information_schema`](information-schema.html) and `pg_catalog` schemas.
+
+For example, the [`information_schema.tables`](information-schema.html#tables) table includes information about all tables in all schemas in all databases, including temp tables:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM information_schema.tables WHERE table_schema='pg_temp_1584540651942455000_1';
+~~~
+
+~~~
+  table_catalog |         table_schema          | table_name |   table_type    | is_insertable_into | version
+----------------+-------------------------------+------------+-----------------+--------------------+----------
+  defaultdb     | pg_temp_1584540651942455000_1 | users      | LOCAL TEMPORARY | YES                |       1
+  defaultdb     | pg_temp_1584540651942455000_1 | vehicles   | LOCAL TEMPORARY | YES                |       1
+(2 rows)
+~~~
+
+If you end the session, all temp tables are lost.
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW session_id;
+~~~
+
+~~~
+             session_id
+------------------------------------
+  15fd69f9831c1ed00000000000000001
+(1 row)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CANCEL SESSION '15fd69f9831c1ed00000000000000001';
+~~~
+
+~~~
+ERROR: driver: bad connection
+warning: connection lost!
+opening new connection: all session settings will be lost
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CREATE TABLE users;
+~~~
+
+~~~
+ERROR: relation "users" does not exist
+SQLSTATE: 42P01
+~~~
+
+## See also
+
+- [`CREATE TABLE`](create-table.html)
+- [`CREATE TABLE AS`](create-table-as.html)
+- [`SHOW CREATE TABLE`](show-create-table.html)
```

Comment:
I might also link to the temp views and temp sequences docs from down here so the user can go to them next if they read to this point.